use-cases/vehicle-routing-time-windows/src/main/resources/application.properties

```diff
@@ -0,0 +1,36 @@
+########################
+# General properties
+########################
+# Enable CORS for runQuickstartsFromSource.sh
+quarkus.http.cors=true
+quarkus.http.cors.origins=/http://localhost:.*/
+# Allow all origins in dev-mode
+%dev.quarkus.http.cors.origins=/.*/
+# Enable Swagger UI also in the native mode
+quarkus.swagger-ui.always-include=true
+########################
+# Timefold properties
+########################
+# TODO: remove after https://github.com/TimefoldAI/timefold-solver/issues/117 is fixed
+quarkus.timefold.solver.domain-access-type=REFLECTION
+# The solver runs for 30 seconds. To run for 5 minutes use "5m" and for 2 hours use "2h".
+quarkus.timefold.solver.termination.spent-limit=30s
```

Comment:
(code style) Not consistent with the other quickstarts: those use white lines to make it a bit more readable